### DISCUSSION TITLE
<title>tcno

### DISCUSSION THREAD

**Discussion by robnjess** [community/community#40224] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Please enter Epic Games's directory, as such: C:\Program Files\Epic Games idk why im getting this in the video online dose not say how to fix this in red say epicgameslauncher,exe not found and i find it and still dose nothing

**Comment by Akash1134** [comment]
Hi @robnjess ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by robnjess** [reply]
ok thank you im sorry
